source/game.screen.stationmap.bmx

```diff
@@ -2778,7 +2778,13 @@ endrem
 
 		'buying stations using the mouse
 		'1. searching
+		GetCurrentPlayer().setHotKeysEnabled(True)
 		If actionMode = MODE_BUY_ANTENNA
+			GetCurrentPlayer().setHotKeysEnabled(False)
```

Comment:
Fehlt hier nicht das "True" ? Also wird garantiert, dass die Zeile oben drueber (`GetCurrentPlayer().setHotKeysEnabled(True)`) danach nochmal aufgerufen wird Eventuel? Wenn nicht, bleiben die devkeys ja ausgeschalten.

Reply:
Soweit ich das sehe, wird onUpdateStationMap praktisch permanent aufgerufen (print eingefügt). Wenn man den Kauf abbricht, ist man noch immer auf der Karte und kommt nicht in den deaktivierenden Zweig. Ich konnte lokal jedenfalls keine Probleme feststellen.